.gitattributes

```diff
@@ -1 +1,2 @@
-* text=auto
+# Enforce Unix newlines
```

Comment:
This needs to be added to the repo manually since grunt-contrib-internal doesn't add it